better-font-awesome.php

```diff
@@ -513,18 +543,67 @@ public function text_callback( $args ) {
      * @return  string  Usage text.
      */
     public function get_usage_text() {
-        return '<div class="bfa-usage-text">' . 
-                __( '<h3>Usage</h3>
-                     <b>Font Awesome version 4.x +</b>&nbsp;&nbsp;&nbsp;<small><a href="http://fontawesome.io/examples/">See all available options &raquo;</a></small><br /><br />
-                     <i class="icon-coffee fa fa-coffee"></i> <code>[icon name="coffee"]</code> or <code>&lt;i class="fa-coffee"&gt;&lt;/i&gt;</code><br /><br />
-                     <i class="icon-coffee fa fa-coffee icon-2x fa-2x"></i> <code>[icon name="coffee" class="fa-2x"]</code> or <code>&lt;i class="fa-coffee fa-2x"&gt;&lt;/i&gt;</code><br /><br />
-                     <i class="icon-coffee fa fa-coffee icon-2x fa-2x icon-rotate-90 fa-rotate-90"></i> <code>[icon name="coffee" class="fa-2x fa-rotate-90"]</code> or <code>&lt;i class="fa-coffee fa-2x fa-rotate-90"&gt;&lt;/i&gt;</code><br /><br /><br />
-                     <b>Font Awesome version 3.x</b>&nbsp;&nbsp;&nbsp;<small><a href="http://fontawesome.io/3.2.1/examples/">See all available options &raquo;</a></small><br /><br />
-                     <i class="icon-coffee fa fa-coffee"></i> <code>[icon name="coffee"]</code> or <code>&lt;i class="icon-coffee"&gt;&lt;/i&gt;</code><br /><br />
-                     <i class="icon-coffee fa fa-coffee icon-2x fa-2x"></i> <code>[icon name="coffee" class="icon-2x"]</code> or <code>&lt;i class="icon-coffee icon-2x"&gt;&lt;/i&gt;</code><br /><br />
-                     <i class="icon-coffee fa fa-coffee icon-2x fa-2x icon-rotate-90 fa-rotate-90"></i> <code>[icon name="coffee" class="icon-2x icon-rotate-90"]</code> or <code>&lt;i class="icon-coffee icon-2x icon-rotate-90"&gt;&lt;/i&gt;</code>',
-                'better-font-awesome' ) .
-                '</div>';
+
+	    ob_start();
+	    ?>
+
+	    <div class="bfa-usage-text">
+			<h3><?php esc_html_e( 'Usage', 'better-font-awesome' ); ?></h3>
+
+		    <b><?php printf( esc_html_x( 'Font Awesome version %s', 'For version 4.x +', 'better-font-awesome' ), '4.x +' ); ?></b>
+		    <small>
+			    <a href="http://fontawesome.io/examples/">
+				    <?php echo esc_html_x( 'See all available options', 'For version 4.x +', 'better-font-awesome' ); ?> &raquo;
+			    </a>
+		    </small>
+		    <br/><br/>
+
+		    <?php
+		    $code_alternative_str = esc_html_x( 'or', 'Text between two variations of code markup examples', 'better-font-awesome' );
+		    //TODO: should I allow these example codes to be translated or not? Add a WP filter for them instead?
```

Comment:
@tcrsavage also re: `coffee`, should I add a filter to allow developers to change what icon is displayed. Or be strict with it and just prescribe it?